Jenkinsfile

```diff
@@ -1,10 +1,11 @@
-@Library('github.com/fabric8io/fabric8-pipeline-library@master')
+@Library('github.com/pranavgore09/fabric8-pipeline-library@standalone-planner')
```

Comment:
If you're using some custom code, send PR to main repo and switch the DIFF back.

Reply:
done  https://github.com/fabric8io/fabric8-pipeline-library/pull/378